### PR TITLE
[OR-322] - Include Oracle JDBC driver in package

### PR DESCRIPTION
**Pull request by usmansaleem**
-- use Oracle maven repo to downloand and package Oracle JDBC driver

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -10,6 +10,7 @@ version.txt
 
 ### Gradle ###
 .gradle
+gradle.properties
```
Comment (CjHare) [inline .gitignore]: Why is there a `gradle.properties` at the root of `Orion` to now ignore?

Comment (usmansaleem) [reply .gitignore]: `gradle.properties` is used by gradle to read personal project properties which we don't want to commit to git (in our case Oracle maven user and password). Hence, it is in ignore list.

File: docs/development/oracle_jdbc.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Oracle Maven Access
+* Orion bundles Oracle JDBC driver which is downloaded from Oracle Maven repository by Gradle.
+* You need to provide your Oracle account credentials to access Oracle Maven repository.
```
Comment (CjHare) [inline docs/development/oracle_jdbc.md]: The reading of this line suggests this PR is introducing a dependency on having a OTN account to build Orion, is that correct?

Comment (usmansaleem) [reply docs/development/oracle_jdbc.md]: @CjHare  It is partially correct now that I am re-reading it. The Orion would still build without Oracle JDBC if we don't create project properties. It needs to be re-worded as `Orion can optionally bundle JDBC driver...`

Comment (usmansaleem) [reply docs/development/oracle_jdbc.md]: but your point is correct, this PR is introducing a dependency on having an Oracle account in order to bundle Oracle JDBC driver

**Pull request merged**